api/client/builder/types.go

```diff
@@ -346,6 +346,70 @@ func (p *ExecutionPayload) ToProto() (*v1.ExecutionPayload, error) {
 	}, nil
 }
 
+func FromProto(payload *v1.ExecutionPayload) (ExecutionPayload, error) {
+	bFee, err := sszBytesToUint256(payload.BaseFeePerGas)
```

Comment:
Does this take care of endianness?

Reply:
I think it does , everything in these methods are just the inverse of `ToProto`